### PR TITLE
[T4A3][W11-B3]Cherlynn Cha 

### PR DESCRIPTION
**Pull request by Cherlynn**

### CODE REVIEW

**Comment by Cherc** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person;
+
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Contact.java]: You forgot to add the header comment here.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+public class Contact {
+
+    public String value;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Contact.java]: this should be declared `private` for better encapsulation.

File: src/seedu/addressbook/data/person/Person.java [hunk 3]
```diff
@@ -11,15 +11,15 @@
 public class Person implements ReadOnlyPerson {
 
     private Name name;
-    private Phone phone;
+    private Contact phone;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Person.java]: You should not change `Phone` object to `Contact`, this approach is totally wrong. Instead you should still declared it as `Phone`. You can check how one of your teammates did it, #1195.

File: src/seedu/addressbook/data/person/ReadOnlyPerson.java [hunk 4]
```diff
@@ -10,7 +10,7 @@
 public interface ReadOnlyPerson {
 
     Name getName();
-    Phone getPhone();
+    Contact getPhone();
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/ReadOnlyPerson.java]: As said above, no need to change this.

File: src/seedu/addressbook/storage/jaxb/AdaptedPerson.java [hunk 5]
```diff
@@ -98,7 +98,7 @@ public Person toModelType() throws IllegalValueException {
             personTags.add(tag.toModelType());
         }
         final Name name = new Name(this.name);
-        final Phone phone = new Phone(this.phone.value, this.phone.isPrivate);
+        final Contact phone = new Phone(this.phone.value, this.phone.isPrivate);
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/storage/jaxb/AdaptedPerson.java]: As said above, no need to change this.

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments. You did it wrong here, after the inheritance, you need to call the super constructor to initialize the value from the child class constructor, see #1195.

**Pull request closed without merging**